_includes/author.html

```diff
@@ -1,6 +1,8 @@
 {% if include.author %}
-  {% assign author = site.authors[include.author] %}
-  <p>{{ author.name }}</p>
+  {% assign author = site.data.authors[include.author] %}
```

Comment:
Nice!

I didn't know this jekyll`s feature.

It`s great to load data this way. I think we can start breaking our _config.yml even more.